livekit-api/livekit/api/version.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.0.1"
```

Comment:
give it a bump since it's pretty solid?

```suggestion
__version__ = "0.1.0"
```